AudioMessageFeatures/AudioMessageFeatures/MessageViewController.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  ViewController.swift
+//  AudioMessageFeatures
+//
+//  Created by Ezra Black on 5/5/20.
+//  Copyright © 2020 Casanova Studios. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+class RecordingTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var recordingNameLabel: UILabel!
+    @IBOutlet weak var timeRecordedLabel: UILabel!
+    @IBOutlet weak var recordingDurationLabel: UILabel!
+    @IBOutlet weak var playButton: UIButton!
+    @IBOutlet weak var recordButton: UIButton!
+    @IBOutlet weak var deleteButton: UIButton!
+    @IBOutlet weak var timeSlider: UISlider!
+    @IBOutlet weak var audioVisualizer: AudioVisualizer!
+    
+}
```

Comment:
- Generally best to put an extra newline after the import statements; I even like to use two new lines. Helps with readability.
- I'd put the cell in a separate file; especially as you add more to these classes, the file is going to become pretty unwieldy and tough to navigate.